tree/tree/src/TLeafD32.cxx

```diff
@@ -48,7 +48,6 @@ TLeafD32::TLeafD32(TBranch *parent, const char *name, const char *type) : TLeaf(
    fValue = nullptr;
    fPointer = nullptr;
    fElement = nullptr;
-   fTitle = type;
```

Comment:
We probably need to find a way to hold both the necessary information in the title.  I.e. maybe keep something like:
```
[n]/d[0,0,10]
```
It might be as simple as (or maybe not):
```
if (fTitle starts with [..])
   fTitle.Append("/").Append(type);
else
   fTitle = type.
```
However the parsing code in the Streamer need to be adjusted to only pass the part after '/' to the StreamerElement when it is present.